src/BuildIt.Forms/BuildIt.Forms.Controls/Platforms/Android/Renderers/PullToRefreshControlRenderer.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.ComponentModel;
+using System.Linq;
+using Android.Content;
+using Android.Views;
+using BuildIt.Forms.Controls;
+using BuildIt.Forms.Controls.Platforms.Android.Renderers;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+using View = Android.Views.View;
+
+[assembly: ExportRenderer(typeof(PullToRefreshControl), typeof(PullToRefreshControlRenderer))]
+namespace BuildIt.Forms.Controls.Platforms.Android.Renderers
+{
+    public class PullToRefreshControlRenderer : Xamarin.Forms.Platform.Android.AppCompat.ViewRenderer<PullToRefreshControl, View>
+    {
+        private const float MaxSwipeDistanceFactor = .6f;
+        private const int RefreshTriggerDistance = 120;
+
+        private MotionEvent downEvent;
+        private int currentPercentage;
+        private float previousY;
+        private readonly int touchSlop;
+        private int? distanceToTriggerRefresh;
+        private bool isAnimatingAfterSuccessfulPulToRefresh;
+
+        public PullToRefreshControlRenderer(Context context)
+            : base(context)
+        {
+            touchSlop = ViewConfiguration.Get(context).ScaledPagingTouchSlop;
+        }
+
+        protected override void OnElementChanged(ElementChangedEventArgs<PullToRefreshControl> e)
+        {
+            base.OnElementChanged(e);
+
+            if (e.OldElement != null)
+            {
+                // Unsubscribe
+            }
+
+            if (e.NewElement != null)
+            {
+                if (!distanceToTriggerRefresh.HasValue)
+                {
+                    if (Element.Height > 0)
+                    {
+                        distanceToTriggerRefresh = (int)Math.Min(Element.Height * MaxSwipeDistanceFactor, RefreshTriggerDistance * Resources.DisplayMetrics.Density);
+                    }
+                }
+
+                //var pullToRefreshControl = new PullToRefreshControl(context);
+                //SetNativeControl(pullToRefreshControl);
+            }
+        }
+
+        protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
+        {
+            base.OnMeasure(widthMeasureSpec, heightMeasureSpec);
+
+            if (!distanceToTriggerRefresh.HasValue && Element.Height > 0)
+            {
+                distanceToTriggerRefresh = (int)Math.Min(Element.Height * MaxSwipeDistanceFactor, RefreshTriggerDistance * Resources.DisplayMetrics.Density);
+            }
+        }
+
+        public override bool OnInterceptTouchEvent(MotionEvent e)
```

Comment:
Add a comment to this method indicating why it's returning true in the context of intercepting touch events